app/code/Bulbulatory/Recomendations/view/adminhtml/ui_component/bulbulatory_recomendations_recommendations_listing.xml

```diff
@@ -0,0 +1,142 @@
+    xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
+    <argument name="data" xsi:type="array">
+        <item name="js_config" xsi:type="array">
+            <item name="provider" xsi:type="string">
+                bulbulatory_recomendations_recommendations_listing.bulbulatory_recomendations_recommendations_listing_data_source
+            </item>
+            <item name="deps" xsi:type="string">
+                bulbulatory_recomendations_recommendations_listing.bulbulatory_recomendations_recommendations_listing_data_source
+            </item>          
+        </item>
+        <item name="spinner" xsi:type="string">
+            spinner_columns
+        </item>
+    </argument>
+    <dataSource name="nameOfDataSource">
+        <argument name="dataProvider" xsi:type="configurableObject">
+            <argument name="class" xsi:type="string">Magento\Framework\View\Element\UiComponent\DataProvider\DataProvider</argument>
+            <argument name="name" xsi:type="string">bulbulatory_recomendations_recommendations_listing_data_source</argument>
+            <argument name="primaryFieldName" xsi:type="string">recommendation_id</argument>
+            <argument name="requestFieldName" xsi:type="string">recommendation_id</argument>
+            <argument name="data" xsi:type="array">
+                <item name="config" xsi:type="array">
+                    <item name="component" xsi:type="string">Magento_Ui/js/grid/provider</item>
+                    <item name="update_url" xsi:type="url" path="mui/index/render" />
+                    <item name="storageConfig" xsi:type="array">
+                        <item name="indexField" xsi:type="string">id</item>
+                    </item>
+                </item>
+            </argument>
+        </argument>
+    </dataSource>
+    <listingToolbar name="listing_top">
+        <filters name="listing_filters" />
+        <columnsControls name="columns_controls" />
+        <filterSearch name="fulltext" />
+        <paging name="listing_paging" />
+        <exportButton name="export_button" />
+        <!-- <argument name="data" xsi:type="array">
+            <item name="config" xsi:type="array">
+                <item name="sticky" xsi:type="boolean">true</item>
+            </item>
+        </argument> -->
+    </listingToolbar>
+    <columns name="spinner_columns">
+        <!-- <column name="recommendation_id">
+            <argument name="data" xsi:type="array">
+                <item name="config" xsi:type="array">
+                    <item name="filter" xsi:type="string">textRange</item>
+                    <item name="sorting" xsi:type="string">asc</item>
+                    <item name="label" xsi:type="string" translate="true">Recommendation Id</item>
+                </item>
+            </argument>
+        </column> -->
+        <column name="customer_id" class="Bulbulatory\Recomendations\Ui\Component\Listing\Column\RecommenderEmail">
```

Comment:
Kolumna ma nazwę `customer_id`, ale wyświetlany jest w niej email - trochę to nieintuicyjne. Lepiej byłoby kolumnę nazwać `customer_email`, żeby rzeczywiście odpowiadała temu, co w niej jest. Wymagać to będzie też zmiany w https://github.com/K44mil/Academy/pull/3/files#diff-592798128860763bef7da7c480f92ee8R31 - zamiast przypisać pobrany email do pola `customer_id`, trzeba będzie przypisać go do pola `customer_email` i wtedy wyświetli się w tej kolumnie.